.github/workflows/ci.yml

```diff
@@ -51,7 +51,7 @@ jobs:
 
     strategy:
       matrix:
-        ember-version: [default, release, beta, canary, lts-3.12, lts-3.16]
+        ember-version: [default, release, beta, canary, lts-3.28, lts-3.24]
```

Comment:
Was dropping support tied to the requirements for the `@cached` decorator, or is it just because/part of the blueprint update? (i.e. are we able to keep the older Ember versions if we wanted to?)

Reply:
I can give it a go -- I was mainly going with the ember support policy, cause I don't know what the glimmer one is

Reply:
i guess it's fine :upside_down_face: 
(assuming we trust the tests)

Reply:
Sorry, I wasn't implying that we should do it one way or the other, just wanted to know if it was deliberate/required or out of convince (what could be fine). Although, now that you proved that it works (at least according to the tests), if it doesn't end up hurting anything, I personally would be inclined to keep the support to aid adoption (though we should also include 3.20 in the matrix in that case).

However, I do think it is important to double check and make sure we in fact could support the `@cached` decorator on all of these versions, at least before we release 2.0 final.